src/pages/home/report/ReportActionItemParentAction.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import OfflineWithFeedback from '../../../components/OfflineWithFeedback';
+import * as Report from '../../../libs/actions/Report';
+import reportPropTypes from '../../reportPropTypes';
+import * as StyleUtils from '../../../styles/StyleUtils';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
+import compose from '../../../libs/compose';
+import withLocalize from '../../../components/withLocalize';
+import ReportActionItem from './ReportActionItem';
+import reportActionPropTypes from './reportActionPropTypes';
+
+const propTypes = {
+    /** The id of the report */
+    reportID: PropTypes.string.isRequired,
+
+    /** The id of the parent report */
+    // eslint-disable-next-line react/no-unused-prop-types
+    parentReportID: PropTypes.string.isRequired,
+
+    /** ONYX PROPS */
+
+    /** The report currently being looked at */
+    report: reportPropTypes,
+
+    /** The actions from the parent report */
+    // TO DO: Replace with HOC https://github.com/Expensify/App/issues/18769.
+    parentReportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
+
+    ...windowDimensionsPropTypes,
+};
+const defaultProps = {
+    report: {},
+    parentReportActions: {},
+};
+
+const ReportActionItemParentAction = (props) => {
+    const parentReportAction = props.parentReportActions[`${props.report.parentReportActionID}`];
+    return (
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(props.report, 'pendingFields.createChat')}
+            errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom') || lodashGet(props.report, 'errorFields.createChat')}
+            errorRowStyles={[styles.ml10, styles.mr2]}
+            onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
+        >
+            <View style={StyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth)}>
+                <View style={[styles.p5, StyleUtils.getReportWelcomeTopMarginStyle(props.isSmallScreenWidth)]} />
+                {parentReportAction && (
+                    <ReportActionItem
+                        report={props.report}
+                        action={parentReportAction}
+                        displayAsGroup={false}
+                        isMostRecentIOUReportAction={false}
+                        shouldDisplayNewMarker={false}
+                        index={0}
+                    />
+                )}
+            </View>
+            <View style={[styles.threadDividerLine]} />
```

Comment:
Coming from https://github.com/Expensify/App/issues/19443

The divider line overlaps with an unread mark, so we should only show the divider line if the first comment in the thread is not marked as unread.